mindjunkies/forums/urls.py

```
@@ -0,0 +1,6 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path("home", views.ForumHomeView.as_view(), name="forum_home")
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)